Services/DataX.Config/DataX.Config/ConfigGeneration/Processor/S600_GenerateJobConfigBatch.cs

```diff
@@ -224,7 +224,7 @@ private async Task<JobConfig> ScheduleSingleJob(JobDeploymentSession job, string
             destFolder = GetJobConfigFilePath(isOneTime, dateString, destFolder);
             var jc = new JobConfig
             {
-                Content = await GetBatchConfigContent(job, defaultJobConfig.ToString(), processStartTime, processEndTime).ConfigureAwait(false),
+                Content = GetBatchConfigContent(job, defaultJobConfig.ToString(), processStartTime, processEndTime),
```

Comment:
Why async is removed?

Reply:
I realized that it doesn't need to be async call.